evm/p2p/kademlia.py

```diff
@@ -189,6 +191,11 @@ def __contains__(self, node):
     def __len__(self):
         return len(self.nodes)
 
+    def __le__(self, other):
+        if not isinstance(other, self.__class__):
+            return super(KBucket, self).__le__(other)
```

Comment:
Can you explain why this isn't an error condition?  If they aren't both `KBuckets` should we be doing comparisons?

Reply:
nope i can't, changed it to raise a TypeError